packages/prover/README.md

```diff
@@ -7,8 +7,6 @@
 
 > This package is part of [ChainSafe's Lodestar](https://lodestar.chainsafe.io) project
 
-Typescript REST client for the [Ethereum Consensus API spec](https://github.com/ethereum/beacon-apis)
```

Comment:
just removing this without a proper replacement doesn' seem right

Reply:
agreed!!